R/environment.R

```diff
@@ -206,6 +194,20 @@ generate_docker_file <- function(custom_docker_image = NULL,
   base_dockerfile <- NULL
   base_dockerfile <- paste0(base_dockerfile, sprintf("FROM %s\n",
                                                      custom_docker_image))
+  
```

Comment:
if no extra packages are specified, can we just use user's custom docker image to start the job without image building?

Reply:
yes, made relevant changes

Reply:
I don't see this get addressed in latest changes. Did I miss something?

Reply:
it is. let's sync offline